### DISCUSSION TITLE
error in CoveragePlot: Error in .Call("CRsparse_colSums", x, na.rm, FALSE, sparseResult) :    "CRsparse_colSums" not resolved from current namespace (Matrix)

### DISCUSSION THREAD

**Discussion by onetaoneryan** [stuart-lab/signac#1550] [Q&A] [answered]
I hope this mail finds you well!
 
I have one question about “CoveragePlot” in signac. One month ago, I could draw the figure  by following code successfully but I don’t know why there are some errors with the same code today. Really appreciate it if you could give me some suggestions.
 
 
library(Signac)
> library(Seurat)
> library(JASPAR2020)
> library(TFBSTools)
> library(ggplot2)
> library(BSgenome.Mmusculus.UCSC.mm10)
> dents.plot2 <- c("Serotonergic neurons")
> DefaultAssay(CON) <- "scATAC"
> DefaultAssay(CFA) <- "scATAC"
> DefaultAssay(TB) <- "scATAC"
> p4 <- CoveragePlot(
+   object = CON,
+   region = "Zbtb16",
+   features = "Zbtb16",
+   idents = dents.plot2,
+   expression.assay = "SCT",
+   ymax =60,
+   extend.upstream = 500,
+   extend.downstream = 10000
+ )
[W::hts_idx_load3] The index file is older than the data file: /Users/rantao/Desktop/RAN/scMultiome_46805/CELLRANGER_scMultiome_46805/1_5210TexAM_CON-1_Multiome_mm/atac_fragments.tsv.gz.tbi
[W::hts_idx_load3] The index file is older than the data file: /Users/rantao/Desktop/RAN/scMultiome_46805/CELLRANGER_scMultiome_46805/2_5210TexAM_CON-2_Multiome_mm/atac_fragments.tsv.gz.tbi
Error in .Call("CRsparse_colSums", x, na.rm, FALSE, sparseResult) :
  "CRsparse_colSums" not resolved from current namespace (Matrix)
 
Best,

Ran

**Comment by timoast** [comment, marked as answer]
This seems to be due to some incompatibilities between CRAN and Bioconductor package versions, please see the solution here: https://github.com/satijalab/seurat/issues/8089#issuecomment-1847816493
